packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -53,12 +54,13 @@ class TextFormField extends FormField<String> {
        assert(maxLines == null || maxLines > 0),
        super(
     key: key,
-    initialValue: controller != null ? controller.value.text : '',
+    initialValue: initialValue ?? '',
```

Comment:
just put the default on the argument, and assert non-null

Reply:
Oops! Done.